core/client/src/main/java/alluxio/client/file/options/OpenFileOptions.java

```diff
@@ -75,17 +75,27 @@ private OpenFileOptions() {
 
   /**
    * @return the location policy used when storing data to Alluxio
+   * @deprecated since version 1.5 and will be removed in version 2.0
    */
   @JsonIgnore
+  @Deprecated
   public FileWriteLocationPolicy getLocationPolicy() {
-    return mLocationPolicy;
+    return mCacheLocationPolicy;
+  }
+
+  /**
+   * @return the location policy used when storing data to Alluxio
+   */
+  @JsonIgnore
+  public FileWriteLocationPolicy getCacheLocationPolicy() {
+    return mCacheLocationPolicy;
   }
 
   /**
    * @return the location policy class used when storing data to Alluxio
    */
-  public String getLocationPolicyClass() {
-    return mLocationPolicy.getClass().getCanonicalName();
+  public String getCacheLocationPolicyClass() {
```

Comment:
this break REST API ... the right way to do it is to keep the old method as well, but deprecate it similar to how your did for `getLocationPolicy`